infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -720,6 +720,40 @@ public void processTopologyChangeNotification(DiscoveryServiceEvent event) {
         }
     }
 
+    /**
+     * Determine if any standby cluster has been added or removed from the topology
```

Comment:
nit -> For correctness, we might need to change this for 'has been added, removed or modified'